Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx

```diff
@@ -71,20 +66,6 @@ class DefaultPagesSettingsPanelBody extends Component {
         }));
     }
 
-    componentDidUpdate(props) {
```

Comment:
Same here, I do not see this reimplemented another way

Reply:
It has been reintroduced and improved in following commits.